djaoapp/static/scss/vendor/bootstrap/_modal.scss

```diff
@@ -235,3 +235,21 @@
   }
 }
 // scss-docs-end modal-fullscreen-loop
+
```

Comment:
Let's move this definition to mallspace.scss (djaoapp-specific custom styles). _modal.scss is part of boostrap scss. When we upgrade bootstrap to a new version, we will forget to add those definitions back.